src/_includes/install/web/deprecated.md

```diff
@@ -0,0 +1,2 @@
+{:.bs-callout-warning}
+The Web Setup Wizard has been deprecated and is no longer supported. It will be removed in Magento 2.4.0. You must use the [command line]({{ site.baseurl }}/guides/v2.3/install-gde/install/cli/install-cli.html) to install Magento aftere the Web Setup Wizard is removed.
```

Comment:
```suggestion
The Web Setup Wizard has been deprecated and is no longer supported. It will be removed in Magento 2.4.0. You must use the [command line]({{ site.baseurl }}/guides/v2.3/install-gde/install/cli/install-cli.html) to install Magento after the Web Setup Wizard is removed.
```

Reply:
Per @chernenm, the upgrade wizard is also being removed. At minimum, these topics need the same or similar notice.

https://devdocs.magento.com/guides/v2.3/comp-mgr/upgrader/upgrade-checklist.html

Also, make sure the messaging is correct. As of 2.3.5, the wizards still work. I'll let Misha comment further.

Reply:
@chernenm, should we amend the statement to include 2.3.6?

For example:

> "It will be removed in 2.3.6 and 2.4.0."

Reply:
Per Misha, it will still be available in 2.3.6. I think the messaging is correct and clear. After it's removed in 2.4.0, you must use the CLI for install and upgrade.

Reply:
I need to figure out how to match two directory paths in template to show the banner message on topic in the upgrade section. It's... not obvious.

Reply:
Do you mean [multiple operators in one 'if'](https://shopify.github.io/liquid/basics/operators/) such as
```
{% if page.url contains "path/to/one/directory" or page.url contains "path/to/another/directory"  %}
```

Reply:
I tried that, but it didn't work.

Reply:
I just noticed an error in the second condition. Running another test with corrected condition.

```
{% if page.url contains "guides/v2.3/install-gde/install/web/" or page.url contains "guides/v2.3/comp-mgr/upgrader/" %}
```

Reply:
That fixed it.

Reply:
I also clarified the deprecation message.